airflow/providers/fab/auth_manager/fab_auth_manager.py

```diff
@@ -147,19 +149,24 @@ def get_cli_commands() -> list[CLICommand]:
             SYNC_PERM_COMMAND,  # not in a command group
         ]
 
-    def get_api_endpoints(self) -> None | Blueprint:
+    def set_api_endpoints(self, connexion_app: connexion.FlaskApp) -> None:
         folder = Path(__file__).parents[0].resolve()  # this is airflow/auth/managers/fab/
         with folder.joinpath("openapi", "v1.yaml").open() as f:
             specification = safe_load(f)
-        return FlaskApi(
+
+        swagger_ui_options = SwaggerUIOptions(
```

Comment:
@RobbeSneyders -> here we switched back to the swagger UI that is built-in (we used to have an external one) - I guess you are committed to update the swagger in the Connexion 3 as it gets released (one of the main reason we used external swagger was that the connexion 2 embedded version at the time had security issue).

Reply:
Thanks! Resolving then.